ci/ubuntu.dockerfile

```diff
@@ -7,6 +7,21 @@ ARG venv=venv
 
 RUN echo "To invalidate cache"
 
+RUN wget http://archive.ubuntu.com/ubuntu/pool/main/o/openssl1.0/libssl1.0.0_1.0.2n-1ubuntu5.3_amd64.deb  && \
+    dpkg -i libssl1.0.0_1.0.2n-1ubuntu5.3_amd64.deb 
+
+WORKDIR /root
+
+RUN cd /usr/lib/x86_64-linux-gnu \
+    && ln -s libssl.so.1.0.0 libssl.so.10 \
+    && ln -s libcrypto.so.1.0.0 libcrypto.so.10 \
+    && curl -fsSL https://github.com/jedisct1/libsodium/archive/1.0.18.tar.gz | tar -xz \
+    && cd libsodium-1.0.18 \
+    && ./autogen.sh \
+    && ./configure \
+    && make install \
+    && cd .. \  
+    && rm -rf libsodium-1.0.18 
```

Comment:
Why add all this stuff? This is a Ubuntu 16.04 docker image, and it doesn't have any problems with libssl1.0 or sodium, so I don't believe this is actually needed. And I guess it will skyrocket CI build time as well.